ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/WebSocketTest.kt

```diff
@@ -74,6 +80,71 @@ class WebSocketTest : ClientLoader() {
         }
     }
 
+    @Test
+    fun testEchoWSS() = clientTests(ENGINES_WITHOUT_WEBSOCKETS + "Js" + "native:CIO") {
+        config {
+            install(WebSockets)
+        }
+
+        test { client ->
+            client.webSocket("wss://echo.websocket.org") {
+                outgoing.send(Frame.Text("PING"))
+                val frame = incoming.receive()
+                assertTrue(frame is Frame.Text)
+                assertEquals("PING", frame.readText())
+            }
+        }
+    }
+
+    @Test
+    fun testWsHandshakeHeaders() {
+        if (PlatformUtils.IS_BROWSER) return // browser websocket client does not support custom headers so the test gets ignored
+        clientTests(ENGINES_WITHOUT_WEBSOCKETS + "native:CIO") {
```

Comment:
you need `return clientTests...` see https://youtrack.jetbrains.com/issue/KT-22228